fluster/main.py

```diff
@@ -37,7 +37,6 @@ class Main:
     # pylint: disable=broad-except
 
     def __init__(self):
-        self.test_suites_dir = TEST_SUITES_DIR
```

Comment:
Why did you remove this line?

Reply:
It's not needed. It should have been deleted in commit https://github.com/fluendo/fluster/commit/9a1a8e7027c969dad8a5983225d9d2fd8e3f44c5

Reply:
I see. If you had done a separated commit specifying that, it would have been a no-brainer